### PR TITLE
More tests

### PR DESCRIPTION
**Pull request by JakeRoggenbuck**

### CODE REVIEW

**Comment by JakeRoggenbuck** [comment]
Waiting on #453

File: src/cpu.rs [hunk 1]
```diff
@@ -214,6 +215,19 @@ impl Speed for CPU {
...
+            name: "cpu".to_string(),
+            number: 0,
+            max_freq: 0,
+            min_freq: 0,
+            cur_freq: 0,
+            cur_temp: 0,
+            cur_usage: 0.0,
+            gov: "powersave".to_string(),
+        }
+    }
```
Comment (Camerooooon) [inline src/cpu.rs:229]: This is what the default method is for.

Comment (Camerooooon) [reply src/cpu.rs:229]: We also have CPU::random();

Comment (JakeRoggenbuck) [reply src/cpu.rs:229]: > This is what the default method is for.

I'll rename this then

**Review by Camerooooon — APPROVED** [review]
:rhinoceros:

**Pull request merged**